### DISCUSSION TITLE
Receiving alerts at the wrong times

### DISCUSSION THREAD

**Discussion by tshayan** [jertel/elastalert2#538] [Q&A] [answered]
Hello,

I live in the EST timezone and I have set elastalert host's timezone to EST as well. My alert schedule looks like this:
```
aggregation:
  schedule: '0 */3 * * *'
```
Based on that, I'd expect to receive alerts every 3 hours at 9am, 12pm, 3pm, 6pm (EST) and so on... Instead the alerts are coming in at 8am, 11am, 2pm, etc which is off by 1 hour every time.

Looking at the log, I see:
`next alert at 2021-11-03 15:00:00+00:00`

Wondering if there is a way to offset the above somewhere in the config so that I can receive alerts at correct times?

**Comment by jertel** [comment, marked as answer]
Options:

`59 */3 * * *` will fire 1 minute before 9am, 12pm, 3pm, etc.

`0 1-23/3 * * * ` will fire starting at 1am, 4am, 9am, 12pm, etc

**Reply by tshayan** [reply]
I thought about those options but figured there may be a way to set it in the configuration. Guess I'll play with the scheduler to make it work. Thanks.
